app/ota_client.py

```diff
@@ -1167,65 +1203,55 @@ def update(self, ecu_update_info):
         # set 'METADATA' state
```

Comment:
`self._ota_status.set_ota_status(OtaStatusString.NORMAL_STATE)`
should be `self.set_ota_status(OtaStatusString.NORMAL_STATE)`